### PR TITLE
add headlamp to list of extensions

### PR DESCRIPTION
**Pull request by yolossn**

### CODE REVIEW

File: api/extensions.json [hunk 1]
```diff
@@ -634,6 +634,40 @@
...
+    {
+      "publisher": {
+        "publisherName": "headlamp",
+        "displayName": "Headlamp"
+      },
+      "extensionName": "Headlamp",
+      "displayName": "Headlamp",
+      "shortDescription": "An easy-to-use and extensible web UI for Kubernetes.",
+      "license": "Apache-2.0",
+      "categories": ["Kubernetes"],
```
Comment (benoitf) [inline api/extensions.json:647]: I think we would need to add 'Docker Desktop' as the category

**Review by benoitf — COMMENTED** [review]
hello @yolossn and thanks for your contribution

do you have tested the extension within Podman Desktop ?

File: api/extensions.json [hunk 2]
```diff
@@ -634,6 +634,40 @@
...
+      "versions": [
+        {
+          "version": "0.23.1",
+          "preview": false,
+          "lastUpdated": "2024-04-23T00:00:00Z",
+          "ociUri": "docker.io/headlamp/headlamp-docker-extension:0.23.1",
+          "files": [
+            {
+              "assetType": "icon",
+              "data": "https://raw.githubusercontent.com/headlamp-k8s/headlamp/main/frontend/public/apple-touch-icon.png"
```
[the remark this answers is not in the document]
Comment (yolossn) [reply api/extensions.json:658]: so should I host these files some where and use that link here?

**Comment by yolossn** [comment]
gentle ping @benoitf

**Comment by benoitf** [comment]
@yolossn I saw you pinged me but AFAIK you've not replied if you tested and how you tested it in Podman Desktop ?

**Comment by benoitf** [comment]
thanks @yolossn I'll merge the PR in 7 days

**Comment by yolossn** [comment]
@benoitf May I know if Headlamp will be listed in Podman Desktop after the PR is merged or will it appear only from the next release.

File: api/extensions.json [hunk 3]
```diff
@@ -634,6 +634,40 @@
...
+      "shortDescription": "An easy-to-use and extensible web UI for Kubernetes.",
+      "license": "Apache-2.0",
+      "categories": ["Kubernetes", "Docker Desktop"],
+      "keywords": ["headlamp", "kubernetes", "dashboard"],
+      "versions": [
+        {
+          "version": "0.23.1",
+          "preview": false,
+          "lastUpdated": "2024-04-23T00:00:00Z",
+          "ociUri": "docker.io/headlamp/headlamp-docker-extension:0.23.1",
```
Comment (benoitf) [inline api/extensions.json:654]:
```suggestion
          "ociUri": "index.docker.io/headlamp/headlamp-docker-extension:0.23.1",
```

**Review by benoitf — APPROVED** [review]

**Comment by benoitf** [comment]
oops forgot to merge it !

**Pull request merged**